pipeline/config/exoscale/sc-config.yaml

```diff
@@ -607,3 +607,6 @@ vulnerabilityExporter:
   resources: {}
   tolerations: []
   affinity: {}
+monitoring:
+  rook:
+    enabled: false
```

Comment:
Maybe we should have this enabled, just to see if something breaks

Reply:
Right the pipeline runs on exoscale and uses rook